lntest/fee_service.go

```diff
@@ -42,6 +43,9 @@ const (
 	// DefaultFeeRateSatPerKw specifies the default fee rate used in the
 	// tests.
 	DefaultFeeRateSatPerKw = 12500
+
+	// httpReadHeaderTimeout sets a timeout for the HTTP server.
+	httpReadHeaderTimeout = lnd.DefaultHTTPHeaderTimeout
```

Comment:
This can now be removed to make the linter happy.

Reply:
Done.